javaCall.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+java -classpath bin/ojdbc6_g.jar:build/libs/catalog-api-test-all.jar edu.oregonstate.mist.catalogapitest.CatalogAPITestApplication server configuration.yaml
```

Comment:
in the future you will be able to simplify things by updating your run script as in the skeleton:
https://github.com/osu-mist/web-api-skeleton/pull/7/files